app/src/main/java/by/paranoidandroid/notesapp/database/AppDatabase.java

```diff
@@ -26,7 +26,7 @@ public static AppDatabase getAppDatabase(Context context) {
                             // Allow queries on the main thread.
                             // Don't do this on a real app!
                             // Only for testing purposes.
-                            .allowMainThreadQueries()
+                            //.allowMainThreadQueries()
```

Comment:
remove it

Reply:
Fixed in https://github.com/Kalyanova/NotesApp/pull/3.